pennylane/operation.py

```diff
@@ -539,6 +539,16 @@ def grad_method(self):
         s_1]=[-1/2, 1, -\pi/2]` is assumed for every parameter.
     """
 
+    # Attributes for compilation transforms
+    is_self_inverse = None
+    """bool: True if the operation is its own inverse."""
+
+    is_symmetric_over_wires = None
+    """bool: True if the operation is the same if you exchange the order of wires."""
+
+    is_composable_rotation = None
+    """bool: True if the composing the operations results in an addition of parameters."""
```

Comment:
It might be good to have `bool or None`, and to specify what `None` means?

Reply:
Oh, and `is_symmetric_over_wires` and `is_composable_rotation` might benefit from a more detailed description/small example?

Reply:
After seeing that `CZ` is symmetric-over-wires, I get it now 😆